extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,272 @@ limitations under the License.
-[/tip]
+### Configure and load pages
 
-### Analytics triggers
+Specify pages in a JSON configuration. Load the JSON configuration from a remote URL with the `src` attribute, or inline it within a `<script>` child element of `<amp-next-page>`. You may choose to both specify a remote URL and inline the JSON object in order to speed up the initial loading of suggestions. In case the `deep-parsing` attribute is present, `<amp-next-page>` will also recursively read suggestions from each loaded document's `<amp-next-page>` element. The component automatically handles loading and unloading the documents from memory for a smooth user experience.
 
-Partial support for analytics is included through the initial host page via two separate events. These are triggered by `<amp-next-page>` and you can track them in your [amp-analytics](https://amp.dev/documentation/components/amp-analytics) config:
+Documents append to the end of the current document as a child of the `<amp-next-page>` element. To prevent shifting page content down, this component must be the last child of the document `<body>`. If needed, any footer content should be embedded inside the `<amp-next-page>` tag and will be displayed once no more article suggestions are available.
 
-| Event                  | Fired when                                               |
-| ---------------------- | -------------------------------------------------------- |
-| `amp-next-page-scroll` | The user scrolls to a new page                           |
-| `amp-next-page-click`  | The user click on an article from the recommendation box |
+#### Inline Configuration
 
-Both of the `triggers` provide the variables `fromUrl` and `toUrl` referring to the previous and current pages. They can be used as follows:
+The inline configuration defines the documents recommended by `amp-next-page` to
+the user as an ordered array of page objects in JSON format.
 
-```html
-<amp-analytics>
-  <script type="application/json">
+Each page object should have the following key/value pairs:
+
+| Key                | Value                                                                                                                                                            |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
+| `title` (required) | String title of the page, will be used when rendering the recommendation box.                                                                                    |
+| `image` (required) | String URL of the image to display in the recommendation box.                                                                                                    |
```

Comment:
Question: Do we have a screenshot of the "recommendation box"? What type of image should it be? Any size dimensions?

Reply:
This is what the recommendation box looks like so it's a square but also it's stylable so we can't really recommend the size
![image](https://user-images.githubusercontent.com/591655/75291131-7d002200-57d6-11ea-9193-455238b80d2a.png)